modules/objdetect/include/opencv2/objdetect/graphical_code_detector.hpp

```diff
@@ -73,6 +73,13 @@ class CV_EXPORTS_W_SIMPLE GraphicalCodeDetector {
     */
     CV_WRAP bool detectAndDecodeMulti(InputArray img, CV_OUT std::vector<std::string>& decoded_info, OutputArray points = noArray(),
                                       OutputArrayOfArrays straight_code = noArray()) const;
+
+#ifdef OPENCV_BINDINGS_PARSER
+    CV_WRAP_AS(detectAndDecodeBytes) NativeByteArray detectAndDecode(InputArray img, OutputArray points = noArray(),
+                                                                     OutputArray straight_code = noArray()) const;
+    CV_WRAP_AS(decodeBytes) NativeByteArray decode(InputArray img, InputArray points, OutputArray straight_code = noArray()) const;
+#endif
```

Comment:
Java bindings use just cv::Mat for it. There is MatOfBytes specialization. M.b. reuse it too?

Reply:
In such way user should call an extra method:
```java
MatOfByte output = detector.detectAndDecodeBytes(qrcode);
new String(output.toArray(), "Shift_JIS"));
```